nixos/modules/services/monitoring/monit.nix

```diff
@@ -36,11 +36,16 @@ in
     ];
 
     systemd.services.monit = {
-      description = "Monit system watcher";
-      after = [ "network-interfaces.target" ];
+      description = "Pro-active monitoring utility for unix systems";
```

Comment:
Sorry, I don't understand the problem? Is it that it doesn't mention the name of the unit being started?